spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/ContainerControllerHelperService.java

```diff
@@ -430,7 +432,7 @@ public JSONObject changeIlluminaLane(HttpSession session, JSONObject json) {
     int container = json.getInt("container");
     StringBuilder b = new StringBuilder();
     b.append(
-        "<i class='italicInfo'>Click in a partition box to beep/type in barcodes, or double click a pool on the right to sequentially add pools to the container</i>");
+        "<i class='italicInfo'>Click in a lane box to beep/type in barcodes, or double click a pool on the right to sequentially add pools to the container</i>");
```

Comment:
When asked about it, Karen shrugged and said that since there's only one lane/partition per SMRT Cell, it didn't really matter to her.